src/settings.ts

```diff
@@ -754,6 +754,7 @@ export const DEFAULT_JAZZBAR_TABLES_NUMBER = 12;
 export const DEFAULT_CONVERSATION_SPACE_TABLES_NUMBER = 10;
 
 export const CHATBOX_NEXT_RENDER_SIZE = 50;
+export const PRIVATE_CHAT_NEXT_RENDER_SIZE = 50;
```

Comment:
I'd call this differently, e.g. `PRIVATE_CHAT_SCROLL_PAGE_SIZE` but I wanted to keep the existing convention